ext/integrations/integrations.c

```diff
@@ -223,8 +223,11 @@ void ddtrace_integrations_minit(void) {
     DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_SYMFONY, "Symfony\\Component\\HttpKernel\\HttpKernel", "__construct",
                                          "DDTrace\\Integrations\\Symfony\\SymfonyIntegration");
 
+    DD_SET_UP_DEFERRED_LOADING_BY_FUNCTION(DDTRACE_INTEGRATION_SQLSRV, "sqlsrv_connect",
+                                         "DDTrace\\Integrations\\SQLSRV\\SQLSRVIntegration");
+
     DD_SET_UP_DEFERRED_LOADING_BY_FUNCTION(DDTRACE_INTEGRATION_WORDPRESS, "wp_check_php_mysql_versions",
-                                           "DDTrace\\Integrations\\WordPress\\WordPressIntegration");
+                                         "DDTrace\\Integrations\\WordPress\\WordPressIntegration");
```

Comment:
It's aligned with the (, not with the offset where the other lines have their " :-D